### DISCUSSION TITLE
ID: Value has to be unique

### DISCUSSION THREAD

**Discussion by daltonsutton** [directus/directus#13678] [Archive]
I'm having an issue after migrating from local to production. I copied all the database tables as is, and I'm unable to post on any collection in production without getting this error. I have the only account, and it's set to admin.

The following fields have invalid values:
ID: Value has to be unique

<img width="833" alt="image" src="https://user-images.githubusercontent.com/3005706/171520729-191e8731-77e1-46c7-bcf1-78aaf93f5db4.png">

I used the DigitalOcean one click install. Everything else seems to be working fine. 

What can I do to fix this? Thanks for any help in advance!

**Comment by azrikahar** [comment]
by local to production, was it PostgreSQL to PostgreSQL or different database vendors? One possibility may be the sequence was not properly reset after the "copy". 

Perhaps you can try the approach shared here: https://github.com/directus/directus/issues/6321#issuecomment-869349722 (note that this may not work with db vendors other than PostgreSQL)

**Reply by daltonsutton** [reply]
Thanks, @azrikahar! I am using PostgreSQL. This pointed me in the right direction and I was able to use the steps listed [here](https://stackoverflow.com/a/66982974) to help fix this.

Thanks again!
